lib/pages/saved_places.dart

```diff
@@ -0,0 +1,375 @@
+      drawer: TrufiDrawer(SavedPlacesPage.route),
+    );
+  }
+
+  Widget _buildAppBar(BuildContext context) {
+    final TrufiLocalization localization =
+        TrufiLocalizations.of(context).localization;
+    return AppBar(title: Text(localization.menuYourPlaces()));
+  }
+
+  Widget _buildBody(BuildContext context) {
+    return Container(
+      padding: const EdgeInsets.all(16.0),
+      child: Column(
+        children: <Widget>[
+          Expanded(child: _savedPlaces()),
+          _addPlaceBttn(),
+        ],
+      ),
+    );
+  }
+
+  //
+
+  Widget _savedPlaces() {
+    final ThemeData theme = Theme.of(context);
+    final TrufiLocalization localization =
+        TrufiLocalizations.of(context).localization;
+    final SavedLocationsBloc savedLocationsBloc =
+        SavedLocationsBloc.of(context);
+    final List<TrufiLocation> data =
+        savedLocationsBloc.locations.reversed.toList();
+
+    final SliverList saveds = SliverList(
+      delegate: SliverChildBuilderDelegate(
+        (BuildContext context, int index) {
+          final TrufiLocation savedPlace = data[index];
+          return Container(
+            margin: const EdgeInsets.only(bottom: 5),
+            child: RaisedButton(
+              onPressed: () {
+                log('bttn');
+              },
+              child: Row(
+                children: <Widget>[
+                  Container(
+                      margin: const EdgeInsets.only(right: 5),
+                      child: Icon(
+                          _typeToIconData(savedPlace.type) ?? Icons.place)),
+                  Expanded(
+                    child: Text(
+                      savedPlace.description,
+                      style: theme.textTheme.body2,
+                      maxLines: 1,
+                    ),
+                  ),
+                  PopupMenuButton<int>(
+                    itemBuilder: (BuildContext context) => [
+                      PopupMenuItem(
+                        value: 1,
+                        child: Text(localization.savedSectionSetIcon()),
+                      ),
+                      PopupMenuItem(
+                        value: 2,
+                        child: Text(localization.savedSectionSetCoordinate()),
+                      ),
+                      PopupMenuItem(
+                        value: 3,
+                        child: Text(localization.savedSectionRemove()),
+                      ),
+                    ],
+                    onSelected: (int index) async {
+                      if (index == 1) {
+                        _changeIcon(savedPlace);
+                      } else if (index == 2) {
+                        _changeCoordinate(savedPlace);
+                      } else if (index == 3) {
+                        setState(() {
+                          savedLocationsBloc.inRemoveLocation.add(savedPlace);
+                        });
+                      }
+                    },
```

Comment:
I think a Bottom Sheet is working better here
https://api.flutter.dev/flutter/material/BottomSheet-class.html

Also, consider using icons next to the menu item labels